src/Bootstrap/RectorConfigsResolver.php

```diff
@@ -40,6 +40,9 @@ private function resolveRectorRecipeConfig(ArgvInput $argvInput): ?string
         return $rectorRecipeFilePath;
     }
 
+    /**
+     * @return bool|string|null
```

Comment:
the `bool` seems will never returned, as there is `if (! file_exists($configFile)) {` check before `realpath()` check, also, in its return declaration, it only define `?string`